mmv1/products/vertexai/Model.yaml

```diff
@@ -0,0 +1,257 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: Model
+base_url: projects/{{project}}/locations/{{region}}/models
+create_url: projects/{{project}}/locations/{{region}}/models:upload
+self_link: 'projects/{{project}}/locations/{{region}}/models/{{name}}'
```

Comment:
What happens if a user adds a new version of their preexisting model out-of-band?

Reply:
Additional versions won't show up in the `GET` response of the model, so terraform will continue to show the original version.